addons/crm_claim/views/crm_claim_view.xml

```diff
@@ -1,5 +1,5 @@
-<?xml version="1.0"?>
-<openerp>
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
     <data>
 
```

Comment:
Change view name , CRM - Claims Tree -to - crm.claim.tree,
CRM - Claims Form to - crm.claim.form
CRM - Claims Search to- crm.claim.search